samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -107,7 +109,23 @@ def _generate_from_pipeline_template(pipeline_template_dir: Path) -> None:
     pipeline_template: Template = _initialize_pipeline_template(pipeline_template_dir)
     bootstrap_context: Dict = _load_pipeline_bootstrap_context()
     context: Dict = pipeline_template.run_interactive_flows(bootstrap_context)
-    pipeline_template.generate_project(context)
+    with tempfile.TemporaryDirectory() as tempdir:
+        LOG.debug("Generating pipeline files into %s", tempdir)
+        context["outputDir"] = "."  # prevent cookiecutter from generating a sub-folder
+        pipeline_template.generate_project(context, tempdir)
+        _copy_dir_contents_fail_on_exist(tempdir, ".")
```

Comment:
better to use ` os.getcwd()` ?

Reply:
No idea whether either of which is better. I changed the method name and use cwd

Reply:
changing back to "."

I added the feature to print out the list of generated files. `os.getcwd` will print out absolute paths and it looks verbose on a list of file paths.

Reply:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/3804518/120029574-253d0180-bfab-11eb-803e-c3e6b4f71bd7.png">